src/reflux/OrderStore.js

```diff
@@ -0,0 +1,21 @@
+import Reflux from 'reflux';
+import Actions from './Actions';
+
+const INITIAL_STATE = {
+  order: {}
+};
+
+class OrderStore extends Reflux.Store {
+  constructor() {
+    super();
+    this.state = {...INITIAL_STATE};
+    this.listenables = Actions;
+  }
+
+  onSetOrder(order) {
+    if (!order) return;
+    this.setState({ order });
+  }
+}
+
```

Comment:
Add a reset function, and call it from GlobalStore
remember to add action name to Actions.js

This resets the order store when closing the widget